streams/runners/storm/runtime/src/test/java/com/hortonworks/streamline/streams/runtime/AvroStreamsSnapshotDeserializerTest.java

```diff
@@ -66,7 +68,10 @@ public void testAvroPayloadConversions() throws Exception {
 
             new Expectations() {
                 {
-                    mockSchemaRegistryClient.getSchemaMetadataInfo(anyLong);
+                    mockSchemaRegistryClient.getSchemaVersionInfo(withEqual(SCHEMA_ID_VERSION));
```

Comment:
http://jmockit.org/api1x/mockit/Expectations.html#withEqual-T-

According to API spec, seems like just putting desired value just works, which is same behavior for Mockito.when(). (`all actual values` or `all matchers`)